core/module/refinements_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../spec_helper'
+
+describe "Module#refinements" do
+  ruby_version_is "3.2" do
+    it "returns list of classes redefined in module" do
+      refined_integer = nil
+      refined_string = nil
```

Comment:
I would propose using `ScratchPad` instead of local variables. It's a bit shorter and more convenient.

Example of its usage:

```ruby
# language/next_spec.rb
ScratchPad.record []
 
[1].each do |i|
  begin
    ScratchPad << :begin
    next
  ensure
    ScratchPad << :ensure
  end
end
 
ScratchPad.recorded.should == [:begin, :ensure]
```